static/manifests/calico/DaemonSet/calico-node.yaml

```diff
@@ -231,6 +231,10 @@ spec:
               value: "true"
             - name: FELIX_PROMETHEUSMETRICSENABLED
               value: "true"
+            {{ range $name, $value := .EnvVars }}
```

Comment:
> Variables below will overwrite the ones specified above them

This one is correct. I've added a comment. Covering it in the inttests will be a bit tricky since we need to check variables inside the container. Also, it's a Kubernetes feature, so I'm not sure _we_ should test it.

Reply:
We are responsible to keep the correct order for the behavior that k0s desires. But a comment in here would do, I guess.

Reply:
:+1: